csrc/python_frontend/fusion_definition.cpp

```diff
@@ -87,10 +87,6 @@ void FusionDefinition::finalizeDefinition() {
     }
 
     buildFusionIr(preschedFusion());
-
-    if (isDebugDumpEnabled(DebugDumpOption::FusionIrPresched)) {
```

Comment:
This prints the original unscheduled fusion generated by the python frontend.

Reply:
Perhaps this should be `DebugDumpOption::FusionIrOriginal`

Reply:
I see -- the word "schedule" is overloaded. How about printing `FusionIrOriginal` inside `FusionExecutorCache::runFusionWithInputs`? AFAICT from [code](https://github.com/NVIDIA/Fuser/blob/78343694da4c7795e035363bfbe867d10a3a327a/csrc/python_frontend/fusion_definition.cpp#L186), both Python and C++ go through that method. So this will reduce divergence between the two APIs.

Reply:
Sounds reasonable!